samples/dymaptic.GeoBlazor.Core.Sample.Shared/_Imports.razor

```diff
@@ -25,4 +25,5 @@
 @using Microsoft.JSInterop
 @using System.Net.Http
 @using System.Net.Http.Json
-@using System.Text.Json
\ No newline at end of file
+@using System.Text.Json
+@using dymaptic.GeoBlazor.Core.Components.Renderers.ColorRamps
```

Comment:
It would be nice to put this in alphabetically.

Reply:
cleaned up